src/browser/wpe/opencdm/open_cdm.cpp

```diff
@@ -180,22 +177,21 @@ int OpenCdm::Update(unsigned char* pbResponse, int cbResponse, std::string& resp
   m_cond_var.wait(lck, [=]() { return m_eState != KEY_SESSION_WAITING_FOR_LICENSE; });
   CDM_LOG_LINE("received a lience update, state is now %s", sessionStateToString(m_eState));
 
-  // FIXME: We get into this state when the key status is considered usable, why
+  // FIXME: We get into UPDATE_LICENSE when the key status is considered usable, why
   // UPDATE_LICENSE is a good state to have set is anybody's guess. Mine would be
   // that this state should be called KEY_STATUS_CHANGED. The implication here would
   // be that the right approach is to fill responseMsg with the key status vector, but
   // without a structured codec, that's rather nasty.
-  if (m_eState == KEY_SESSION_UPDATE_LICENSE || m_eState == KEY_SESSION_REMOVED)
-    ret = 0;
-  else if (m_eState == KEY_SESSION_MESSAGE_RECEIVED) {
+  if (m_eState == KEY_SESSION_MESSAGE_RECEIVED) {
     responseMsg.assign("message:");
     responseMsg.append(m_message.c_str(), m_message.length());
     CDM_LOG_LINE("The response message contains %d bytes and its contents are:", responseMsg.size());
     CDMDumpMemory(reinterpret_cast<const uint8_t*>(responseMsg.data()), responseMsg.size());
   }
 
-  CDM_LOG_LINE("return %d from Update", ret);
-  return ret;
+  CDM_LOG_LINE("response message %s", responseMsg.c_str());
```

Comment:
These messages are raw bytes, so using `CDMDumpMemory` would be better, but I have made the logs quite nosy in the past doing these things, because we also dump the bytes in WebKit, but it can be handy to check nothing has gone wrong in the comms.